packages/react-components/lib/themes/global-theme.tsx

```diff
@@ -2,22 +2,24 @@ import { withStyles } from '@material-ui/core/styles';
 
 export const GlobalCss = withStyles((theme) => ({
   '@global': {
+    // override accordian background to distinguish it from onipanel
     '.MuiAccordion-root': {
       backgroundColor: theme.palette.background.default,
     },
+    // override text color of disabled button to make it visible
     '.MuiButtonBase-root.Mui-disabled': {
       color: theme.palette.text.primary,
     },
-    '.MuiAppBar-root': {
-      backgroundColor: theme.palette.primary.main,
-    },
+    // highlight table header in panels
     '.MuiTableCell-head': {
       background: 'rgba(0, 0, 0, 0.1)',
       borderBottom: 'none',
     },
+    // override text color of forms to be visible in theme background colors
     '.MuiFormLabel-root': {
       color: theme.palette.text.primary,
     },
+    // text color of forms to distinguish them when focus
```

Comment:
This and the above form css breaks how form looks.

Without global css
![image](https://user-images.githubusercontent.com/45189696/125218407-83246f00-e2f5-11eb-9d7a-221f6649e3cc.png)

With global css
![image](https://user-images.githubusercontent.com/45189696/125218377-6e47db80-e2f5-11eb-8535-67fc5658c744.png)

It's no longer able to differentiate between the hint text and the actual text.

Even with the global css, forms still look broken in both rmf-light and rmf-dark

![image](https://user-images.githubusercontent.com/45189696/125218550-d4ccf980-e2f5-11eb-8434-3e37bea1c3c6.png)
Buttons color same as background, bad text colors.

![image](https://user-images.githubusercontent.com/45189696/125218585-e57d6f80-e2f5-11eb-8e8b-3ddd80d9fe44.png)
Buttons color same as background, wrong colors some elements (date selectors, text field underlines etc).

The "cancel" and "ok" buttons in date selector becomes completely invisible
![image](https://user-images.githubusercontent.com/45189696/125218987-9f74db80-e2f6-11eb-87c9-e082cdfff3c0.png)